students/yixingxu/lesson02/Fizz-Buzz.py

```diff
@@ -0,0 +1,9 @@
+for i in range(1,100):
```

Comment:
python modules (file names) follow convention of snake_case naming -> `fizz_buzz.py`